src/main/kotlin/com/vanniktech/maven/publish/Configurer.kt

```diff
@@ -0,0 +1,10 @@
+package com.vanniktech.maven.publish
+
+import org.gradle.api.component.SoftwareComponent
+import org.gradle.api.tasks.bundling.AbstractArchiveTask
+
+internal interface Configurer {
```

Comment:
`Uploader`? I don't have a better name either. Let's define the names once we've also got the other implementation sorted out and clean up afterwards